site/en/docs/extensions/mv3/manifest/content_scripts/index.md

```diff
@@ -0,0 +1,262 @@
+### Globs and URL matching examples {: #matching-samples }
+{% endDetailsSummary %}
+
+#### `"include_globs"` {: #incl-globs }
+
+{% Label %}manifest.json{% endLabel %}
+
+```json/5
+{
+  ...
+  "content_scripts": [
+    {
+      "matches": ["https://*.example.com/*"],
+      "include_globs": ["https://???.example.com/foo/"],
+      "js": ["content-script.js"]
+    }
+  ],
+  ...
+}
+```
+
+<div class="switcher">
+{% Compare 'better', 'Matches' %}
+```text
+https://www.example.com/foo/bar
+https://the.example.com/foo/
+```
+{% endCompare %}
+
+{% Compare "worse", "Does not match" %}
+```text
+https://my.example.com/foo/bar
+https://example.com/foo/
+https://www.example.com/foo
+```
+{% endCompare %}
+</div>
+
+{% Label %}manifest.json{% endLabel %}
+
+```json/5
+{
+  ...
+  "content_scripts": [
+    {
+      "matches": ["https://*.example.com/*"],
+      "include_globs": ["*example.com/???s/*"],
+      "js": ["content-script.js"]
+    }
+  ],
+  ...
+}
+```
+
+<div class="switcher">
+{% Compare 'better', 'Matches' %}
+```text
+https://www.example.com/arts/index.html
+https://www.example.com/jobs/index.html
+```
+{% endCompare %}
+
+{% Compare "worse", "Does not match" %}
+```text
+https://www.example.com/sports/index.html
+https://www.example.com/music/index.html
+```
+{% endCompare %}
+</div>
+
+#### `"exclude_globs"` {: #excl-globs }
+
+{% Label %}manifest.json{% endLabel %}
+
+```json/5
+{
+  ...
+  "content_scripts": [
+    {
+      "matches": ["https://*.example.com/*"],
+      "exclude_globs": ["*science*"],
+      "js": ["content-script.js"]
+    }
+  ],
+  ...
+}
+```
+
+<div class="switcher">
+{% Compare 'better', 'Matches' %}
+```text
+https://history.example.com
+https://.example.com/music
+```
+{% endCompare %}
+
+{% Compare "worse", "Does not match" %}
+```text
+https://science.example.com
+https://www.example.com/science
+```
+{% endCompare %}
+</div>
+
+#### Advanced customization example {: #all-custom }
+
+{% Label %}manifest.json{% endLabel %}
+
+```json/5-7
+{
+  ...
+  "content_scripts": [
+    {
+      "matches": ["https://*.example.com/*"],
+      "exclude_matches": ["*://*/*business*"],
+      "include_globs": ["*example.com/???s/*"],
+      "exclude_globs": ["*science*"],
+      "js": ["content-script.js"]
+    }
+  ],
+  ...
+}
+```
+
+<div class="switcher">
+{% Compare 'better', 'Matches' %}
+```text
+https://www.example.com/arts/index.html
+https://.example.com/jobs/index.html
+```
+{% endCompare %}
+
+{% Compare "worse", "Does not match" %}
+```text
+https://science.example.com
+https://www.example.com/jobs/business
+https://www.example.com/science
+```
+{% endCompare %}
+</div>
+
+{% endDetails %}
+
+## Frames {: #frames }
+
+The `"all_frames"` field specifies if the content script should be injected into all frames matching the specified URL requirements. If set to `false` it will only inject into the topmost frame. It can be used along with `"match_about_blank"` to inject into an `about:blank` frame. 
```

Comment:
```suggestion
The `"all_frames"` field specifies if the content script should be injected into all frames matching the specified URL requirements. If set to `"false"` it will only inject into the topmost frame. It can be used along with `"match_about_blank"` to inject into an `about:blank` frame. 
```